wpilibc/athena/src/Utility.cpp

```diff
@@ -35,7 +38,22 @@ bool wpi_assert_impl(bool conditionValue, llvm::StringRef conditionText,
     llvm::raw_svector_ostream locStream(locBuf);
     locStream << funcName << " [";
     llvm::SmallString<128> fileTemp;
+
+#ifdef _WIN32
+  char fname[60];
+  char ext[10];
+  _splitpath_s(fileName.c_str(fileTemp), nullptr, 0, nullptr, 0, fname, 60, ext, 10);
+ locStream << fname << ":" << lineNumber << "]";
+#elif __APPLE__
+  auto file = fileName.c_str(fileTemp);
+  int len = strlen(file) + 1;
+  char* basestr = new char[len + 1];
+  strncpy(basestr, file, len);
+  locStream << basestr << ":" << lineNumber << "]";
+  delete[] basestr;
+#else
     locStream << basename(fileName.c_str(fileTemp)) << ":" << lineNumber << "]";
+#endif
```

Comment:
Ok. I'll create an issue for this in the WPIUtil repo, and probably do it on my local fork setup for the new builds.